deploy/rinkeby/0092_liquidity_rewards.ts

```diff
@@ -0,0 +1,135 @@
+import { ethers as ethershardhat, hardhatArguments } from "hardhat";
+import { deployments } from "hardhat";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+require("dotenv").config();
+
+module.exports = async (hre: HardhatRuntimeEnvironment) => {
+	let initial_run = process.env.INITIAL_RUN == "false" ? true : false;
```

Comment:
Have you added this env var to `env.sample`? I don't see any changes to `env.sample`